src/app/clusters/window-covering-server/window-covering-server.cpp

```diff
@@ -0,0 +1,685 @@
+}
+
+//------------------------------------------------------------------------------
+// TiltActuator
+//------------------------------------------------------------------------------
+
+void WindowCover::TiltActuator::OpenLimitSet(uint16_t limit)
+{
+    writeAttribute(mEndPoint, ZCL_WC_INSTALLED_OPEN_LIMIT_TILT_ATTRIBUTE_ID, static_cast<uint8_t *> &limit, ZCL_INT16U_ATTRIBUTE_TYPE);
+}
+
+uint16_t WindowCover::TiltActuator::OpenLimitGet(void)
+{
+    uint16_t limit = 0;
+    readAttribute(mEndPoint, ZCL_WC_INSTALLED_OPEN_LIMIT_TILT_ATTRIBUTE_ID, static_cast<uint8_t *> &limit, sizeof(uint16_t));
+    return limit;
+}
+
+void WindowCover::TiltActuator::ClosedLimitSet(uint16_t limit)
+{
+    writeAttribute(mEndPoint, ZCL_WC_INSTALLED_CLOSED_LIMIT_TILT_ATTRIBUTE_ID, static_cast<uint8_t *> &limit, ZCL_INT16U_ATTRIBUTE_TYPE);
+}
+
+uint16_t WindowCover::TiltActuator::ClosedLimitGet(void)
+{
+    uint16_t limit = 0;
+    readAttribute(mEndPoint, ZCL_WC_INSTALLED_CLOSED_LIMIT_TILT_ATTRIBUTE_ID, static_cast<uint8_t *> &limit, sizeof(uint16_t));
+    return limit;
+}
+
+void WindowCover::TiltActuator::PositionSet(uint16_t percent100ths)
+{
+    uint8_t percent = static_cast<uint8_t>(percent100ths / 100);
+    uint16_t value  = Percent100thsToValue(percent100ths);
+
+    writeAttribute(mEndPoint, ZCL_WC_CURRENT_POSITION_TILT_ATTRIBUTE_ID,  static_cast<uint8_t *> &value, ZCL_INT16U_ATTRIBUTE_TYPE);
+    writeAttribute(mEndPoint, ZCL_WC_CURRENT_POSITION_TILT_PERCENTAGE_ATTRIBUTE_ID, static_cast<uint8_t *> &percent, ZCL_INT8U_ATTRIBUTE_TYPE);
+    writeAttribute(mEndPoint, ZCL_WC_CURRENT_POSITION_TILT_PERCENT100_THS_ATTRIBUTE_ID, static_cast<uint8_t *> &percent100ths,
+                   ZCL_INT16U_ATTRIBUTE_TYPE);
+    emberAfWindowCoveringClusterPrint("TiltActuator::PositionSet(%u%%)", percent100ths / 100);
+}
+
+uint16_t WindowCover::TiltActuator::PositionGet()
+{
+    uint16_t percent100ths = 0;
+    readAttribute(mEndPoint, ZCL_WC_CURRENT_POSITION_TILT_PERCENT100_THS_ATTRIBUTE_ID, static_cast<uint8_t *> &percent100ths,
+                  sizeof(uint16_t));
+    return percent100ths;
+}
+
+void WindowCover::TiltActuator::TargetSet(uint16_t percent100ths)
+{
+    emberAfWindowCoveringClusterPrint("Actuator::TargetSet(+%u%%)", percent100ths / 100);
+    writeAttribute(mEndPoint, ZCL_WC_TARGET_POSITION_TILT_PERCENT100_THS_ATTRIBUTE_ID, static_cast<uint8_t *> &percent100ths,
+                   ZCL_INT16U_ATTRIBUTE_TYPE);
+}
+
+uint16_t WindowCover::TiltActuator::TargetGet()
+{
+    uint16_t percent100ths = 0;
+    readAttribute(mEndPoint, ZCL_WC_TARGET_POSITION_TILT_PERCENT100_THS_ATTRIBUTE_ID, static_cast<uint8_t *> &percent100ths, sizeof(uint16_t));
+    return percent100ths;
+}
+
+void WindowCover::TiltActuator::NumberOfActuationsIncrement()
+{
+    uint16_t count = static_cast<uint16_t>(NumberOfActuationsGet() + 1);
+    writeAttribute(mEndPoint, ZCL_WC_NUMBER_OF_ACTUATIONS_TILT_ATTRIBUTE_ID, static_cast<uint8_t *> &count, ZCL_INT16U_ATTRIBUTE_TYPE);
```

Comment:
```suggestion
    writeAttribute(mEndPoint, ZCL_WC_NUMBER_OF_ACTUATIONS_TILT_ATTRIBUTE_ID, static_cast<uint8_t *>(&count), ZCL_INT16U_ATTRIBUTE_TYPE);
```

Reply:
Replaced with the corresponding Accessors.h function.